packages/taquito-rpc/src/rpc-client-modules/rpc-cache.ts

```diff
@@ -1165,4 +1170,58 @@ export class RpcClientCache implements RpcClientInterface {
       return response;
     }
   }
+
+  /**
+   *
+   * @param contract address of the contract we want to retrieve ticket balance of
+   * @param ticket object to specify ticketer, content type and content
```

Comment:
I think we can specify it to be 'Ticket token parameter object that contains ticketer, content type, and content'